app/styles/components/mobile-dialog.css

```diff
@@ -0,0 +1,30 @@
+.mobile-dialog {
+    display: block;
+    border-radius: 3px;
```

Comment:
- as the rds design system has a base unit of `4px` please use multiples of 4 here (4, 8, 16, ...)